src/irvisualizer/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "halide-irvisualizer",
+  "description": "Halide IRVisualizer web toolchain",
+  "author": "Halide IRVisualizer workgroup",
+  "version": "0.0.1",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "scripts": {
+    "build": "esbuild StmtToViz_main.js --bundle --minify --sourcemap --outdir=build --target=es6",
+    "lint": "eslint StmtToViz_main.js",
+    "watch": "esbuild StmtToViz_main.js --bundle --watch --outdir=build"
```

Comment:
Old filename.

Reply:
@mcourteaux Done. I highly doubt it will ever tun because `package.json` is a dummy file so far.

@steven-johnson is now resolving the CI issue. So I will refrain from `git push` till he has finished editing the PR.

Reply:
This should be ready for you to resume editing, thanks for your patience.

Reply:
Thanks @steven-johnson . I finished editing. Closing this thread.